websites/D/DCGO/presence.ts

```diff
@@ -0,0 +1,52 @@
+const presence = new Presence({
+	clientId: "1075115687888044072",
+});
+
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/NPVqgsx.png",
+	};
+	if (document.location.pathname.toLowerCase() === "/") {
```

Comment:
As `document.location.pathname` is used many times in this presence, destructure this property for simplicity.
```suggestion
	const { pathname } = document.location;
	if (pathname.toLowerCase() === "/") {
```

Reply:
okey

Reply:
i changed

Reply:
@theusaf